### PR TITLE
feat(console): show the connect status of webconsole

### PR DESCRIPTION
**Pull request by helight**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind feature

**What this PR does / why we need it**:

添加了显示webconsole的连接状态，明确是否可用。

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by huxiaoliang** [comment]
@helight  please update you commit message so that pass message lint check, thanks in advance.

File: web/console/public/webtty.html [hunk 1]
```diff
@@ -63,43 +65,58 @@
...
+      tipsElem.innerText = content
+    }
+
+    setTips("Connecting...")
+
   </script>
 </body>
 
-</html>
+</html>
```
Comment (huxiaoliang) [inline web/console/public/webtty.html:122]: no new line at the end of file

Comment (helight) [reply web/console/public/webtty.html]: ok

**Comment by helight** [comment]
> @helight please update you commit message so that pass message lint check, thanks in advance.
it's ok now。

**Comment by johnny19941216** [comment]
+1

**Pull request merged**